src/app/reference/controllers/benthic-attributes.ctrl.js

```diff
@@ -63,27 +64,64 @@ angular.module('app.reference').controller('BenthicAttributesCtrl', [
             return regions.join(', ') || '-';
           }
         }
-      ]
+      ],
+      toolbar: {
+        template: 'app/reference/partials/fish-families-toolbar.tpl.html',
```

Comment:
`fish-families...` in `benthic-attributes...`?

Reply:
my mistake while copy/paste => changed